keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1336,13 +1318,9 @@ type AllocationResult struct {
 	SharedResource bool
 }
 
-// PodScheduling objects get created by a scheduler when it needs
-// information from resource driver(s) while scheduling a Pod that uses
-// one or more unallocated ResourceClaims with delayed allocation.
-//
-// Alternatively, a scheduler extender might be configured for the
-// resource driver(s). If all drivers have one, this object is not
-// needed.
+// PodScheduling objects get created by a scheduler when it handles
+// a pod which uses one or more unallocated ResourceClaims with delayed
+// allocation.
 type PodScheduling struct {
```

Comment:
Please name this more specifically, e.g. from the name I expected it to be an explanation of the scheduler's decision or something.

Reply:
Any suggestions? I know that the current usage is just for claims, but calling it `PodClaimScheduling` would become inappropriate as soon as we find other usages for this object, which is a possibility.

On the other hand this is only alpha. We can name it `PodClaimScheduling` now and still rename it later.

@thockin : you suggested that name. What do you think?

Reply:
Can we keep this as-is for now and make a final decision during the API review of the implementation?

I don't want to miss the KEP deadline because of a naming discussion.

Reply:
Yes, apis are never final until then.

Is there an explanation in this KEP why the scheduler has to do this reach-out to drivers? I am reading from the top and haven't figured it out yet.